src/geo/ui/infowindow-view.js

```diff
@@ -246,6 +246,16 @@ var Infowindow = View.extend({
     }
   },
 
+  _sanitizeValue: function (key, obj) {
+    var t = typeof (obj);
+
+    if (t !== 'object' || obj === null) {
```

Comment:
Suggestion: we could use [`_.isObject`](http://underscorejs.org/#isObject) and [`isArray`](http://underscorejs.org/#isArray) here... 😄